src/core/src/op/scatter_update.cpp

```diff
@@ -124,3 +124,65 @@ bool op::v3::ScatterUpdate::has_evaluate() const {
     }
     return false;
 }
+
+namespace {
+OPENVINO_SUPPRESS_DEPRECATED_START
+bool scatter_label_evaluator(const Node* node, TensorLabelVector& output_labels) {
+    const auto& input_values = node->input_values();
+    if (input_values.size() < 4) {
+        return false;
+    }
+
+    constexpr auto data_in_idx = 0;
+    constexpr auto updates_in_idx = 2;
+    bool in0_has_no_labels = ov::has_no_labels(input_values[data_in_idx].get_tensor().get_value_label());
+    bool in2_has_no_labels = ov::has_no_labels(input_values[updates_in_idx].get_tensor().get_value_label());
+
+    if (in0_has_no_labels && in2_has_no_labels) {
+        return false;
+    }
+
+    HostTensorVector input_tensors(input_values.size());
+    for (size_t i = 0; i < input_values.size(); ++i) {
+        const auto& input = input_values[i];
+        if (i == data_in_idx || i == updates_in_idx) {
+            auto input_labels = input.get_tensor().get_value_label();
+            if (ov::has_no_labels(input_labels)) {
+                input_labels = std::vector<size_t>{0};
+            }
+            auto labels_constant = op::v0::Constant::create(ov::element::u64, input.get_shape(), input_labels);
```

Comment:
Why not create the `HostTensor` or `ov::Tensor` directly without creating `Constant`?

Reply:
(Change in progress) I considered it too, I will try to update `HostTensor` to `ov::Tensor`.

Reply:
Done